packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -188,13 +192,75 @@ const useTrackStyles = makeStyles({
   }),
 });
 
+/**
+ * Styles for the mark slot
+ */
+const useMarksWrapperStyles = makeStyles({
+  marksWrapper: theme => ({
+    position: 'relative',
+    display: 'grid',
+    outline: 'none',
+    zIndex: '1',
+    whiteSpace: 'nowrap',
+    '& .ms-Slider-mark': {
+      // TODO: theme neutralStrokeOnBrand
+      background: 'white',
+    },
+
+    '& .first, .last': {
+      opacity: '0',
+    },
+  }),
+
+  horizontal: theme => ({
+    marginTop: 'calc(var(--slider-rail-size) + var(--slider-mark-size))',
+    marginLeft: 'calc(var(--slider-thumb-size) / 2)',
+    marginRight: 'calc(var(--slider-thumb-size) / 2)',
+    justifyItems: 'end',
+
+    '& .ms-Slider-markItemContainer': {
+      display: 'flex',
+      flexDirection: 'column',
+      transform: 'translateX(50%)',
+      alignItems: 'center',
+    },
+
+    '& .ms-Slider-mark': {
+      height: '4px',
+      width: '1px',
+    },
+  }),
+
+  vertical: theme => ({
+    marginTop: 'calc(var(--slider-thumb-size) / 2)',
+    marginBottom: 'calc(var(--slider-thumb-size) / 2)',
+    marginLeft: 'calc(var(--slider-rail-size) + var(--slider-mark-size))',
+    justifyItems: 'start',
+
+    '& .ms-Slider-markItemContainer': {
+      display: 'flex',
+      flexDirection: 'row',
+      transform: 'translateY(50%)',
+      alignItems: 'center',
+      maxWidth: '100%',
+      maxHeight: '100%',
+    },
+
+    '& .ms-Slider-mark': {
+      height: '1px',
+      width: 'var(--slider-mark-size)',
+    },
+  }),
+});
+
 /**
  * Styles for the thumb slot
  */
 const useThumbWrapperStyles = makeStyles({
   thumbWrapper: theme => ({
     position: 'absolute',
     outline: 'none',
+    zIndex: '2',
```

Comment:
Not an expert on z-index but my impression was it's generally brittle and should be avoided if possible--might want to get input from @khmakoto or someone else about best practices here